src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -165,6 +165,9 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy {
               } else {
                 this.courseHierarchy = data.courseHierarchy;
               }
+              if (!isSingleContent && _.get(this.courseHierarchy, 'mimeType') !== 'application/vnd.ekstep.content-collection') {
```

Comment:
Write the unit-test cases for cover this scenario.

Reply:
Use 
configService.appConfig.PLAYER_CONFIG.MIME_TYPE.collection
Instead of hardcoding the string